doc/source/user_guide/hpc_slurm.rst

```diff
@@ -0,0 +1,611 @@
+environment which is accessible from the compute nodes.
+
+To do that you can find where your Python distribution is installed using:
+
+.. code-block:: console
+
+    user@machine:~$ which python3
+    /usr/bin/python3
+
+You can print the version of Python you have available by doing:
+
+.. code-block:: console
+
+    user@machine:~$ python3 --version
+    Python 3.9.16
+
+You should be aware that your machine might have installed other Python versions.
+If those installations are already in the ``PATH``, you can use autocomplete (``TAB``)
+to find out:
+
+.. code-block:: console
+
+    user@machine:~$ which python3[TAB]
+    python3             python3-intel64     python3.10-config   python3.11          python3.12          python3.8           python3.8-intel64   python3.9-config  
+    python3-config      python3.10          python3.10-intel64  python3.11-config   python3.12-config   python3.8-config    python3.9 
+    $ which python3.10
+    /usr/bin/python3.10
+
+Remember you should use a Python version which is compatible with PyMAPDL.
+For more information visit :ref:`ref_pymapdl_installation`.
+
+The ``which`` command returns the path where the Python executable is installed.
+You can use that executable to create your own Python virtual environment in a directory
+which is accessible from all the compute nodes.
+For most of HPC cluster, ``/home/$user`` is generally available to all nodes.
+Then the virtual environment can be created in the directory ``/home/user/.venv``:
+
+.. code-block:: console
+
+    user@machine:~$ python3 -m venv /home/user/.venv
+
+Then you can install PyMAPDL after activating the virtual environment:
+
+.. code-block:: console
+
+    user@machine:~$ source /home/user/.venv/bin/activate
+    (.venv) user@machine:~$ pip install ansys-mapdl-core
+    Collecting ansys-mapdl-core
+    Downloading ansys_mapdl_core-0.68.1-py3-none-any.whl (26.9 MB)
+        ━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━ 26.9/26.9 MB 37.3 MB/s eta 0:00:00
+    Collecting pexpect>=4.8.0
+    Using cached pexpect-4.9.0-py2.py3-none-any.whl (63 kB)
+    Collecting click>=8.1.3
+    ...
+
+You can test if this virtual environment is accessible from the compute nodes by
+running the following bash script ``test.sh``:
+
+.. code-block:: bash
+
+    #!/bin/bash
+    #SBATCH --job-name=myjob
+    #SBATCH --nodes=1
+    #SBATCH --ntasks-per-node=4
+    #SBATCH --time=01:00:00
+
+    # Commands to run
+    echo "Testing Python!"
+    source /home/user/.venv/bin/activate
+    python -c "from ansys.mapdl import core;print(f'PyMAPDL version {core.__version__} was successfully imported!')"
+
+using: 
+
+.. code-block:: console
+
+    user@machine:~$ srun test.sh
+
+This command might take around 1-2 minutes to complete depending on the amount of free
+resources available in the cluster.
+The console output should show:
+
+.. code-block:: text
+
+    Testing Python!
+    PyMAPDL version 0.68.1 was successfully imported!
```

Comment:
Technical doc/code should not use (or at least very rarely use) exclamation points. I'd advise changing them to periods.

Reply:
This code is for demostration purposes. But noted.